src/cms/_tests.py

```diff
@@ -4,7 +4,7 @@
 from zipfile import ZipFile
 from bs4 import BeautifulSoup
 
-from django.contrib.auth.models import User
+from django.contrib.auth import get_user_model
```

Comment:
Seeing that you changed the `User` import to `get_user_model`, we should maybe think about following the official Django suggestion of implementing a custom user model to ease later extensibility (see [the official Django documentation](https://docs.djangoproject.com/en/2.2/topics/auth/customizing/#using-a-custom-user-model-when-starting-a-project)) as long as we haven't deployed the project yet and can still easily replace the user model.
Could this maybe also allow us to drop the `UserProfile` model?

Reply:
Yeah, we should consider this option.
My initial thought when creating the UserProfile model was that that we could use the User model solely for authentication purposes and the Profile model for functional extensions. But you're right, most of docs & tutorials seem to recommend a custom user model.